packages/core/types/ts3.6/effects.d.ts

```diff
@@ -381,22 +381,36 @@ export type HelperWorkerParameters<T, Fn extends (...args: any[]) => any> = Last
  * action to the Store. This effect is non-blocking and any errors that are
  * thrown downstream (e.g. in a reducer) will not bubble back into the saga.
  *
- * @param action [see Redux `dispatch` documentation for complete info](https://redux.js.org/api/store#dispatchaction)
+ * @param action [see Redux `dispatch` documentation for complete info](http://redux.js.org/docs/api/Store.html#dispatch)
  */
 export function put<A extends Action>(action: A): PutEffect<A>
 
+/**
+ * Interface for using thunk action from Redux Toolkit in `put`.
+ */
+interface AsyncThunkAction {}
```

Comment:
This is an empty interface - this effectively behaves like `any`, see the playground here:
https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgIIGcCeIEBUAWAriANaoJjAD2IyA3gL4BQTAJhAgDZxQozEVqtAA6EwAHlTIIAD0ghW6NFhwFiZQTQB8ACjBFS5SjQBcaAJRmAblWCsWAegfIAAmHQBaWcI5gvUKCooJlEwHQBGACYAZnMWIA